projects/libvpx/build.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash -ex
+# Copyright 2018 Google Inc.
+# Modifications copyright (C) 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+STANDALONE_MAIN="main.cc"
```

Comment:
Could it be built with those main-stubs in the '/opt' directory in sydr/ubuntu20.04-sydr-fuzz?

Reply:
nope